Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -757,6 +757,37 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .AddParameter("object", _("Object"), "Drawer")
       .AddParameter("expression", _("X drawing position"))
       .AddParameter("expression", _("Y drawing position"));
-
+  obj.AddAction("SetAntialiasing",
+                  _("Set antialiasing on the shape drawer"),
+                  _("(De)Activate antialiasing on the shape drawer"),
+                  _("Set antialiasing of _PARAM0_ to _PARAM1_"),
+                  _("Setup"),
+                  "res/actions/antialiasing24.png",
+                "res/actions/antialiasing .png")
+        .AddParameter("object", _("Object"), "Drawer")
+        .AddParameter("yesorno", _("Activated ?"), "Is the antialiasing activated ?");
+    obj.AddAction("SetAntialiasingQuality",
+                  _("Set antialiasing quality for the shape drawer"),
+                  _("Set antialiasing quality for the shape drawer"),
+                  _("Set antialiasing quality of _PARAM0_ to _PARAM1_"),
+                  _("Setup"),
+                  "res/actions/antialiasing24.png",
+                "res/actions/antialiasing .png")
+        .AddParameter("object", _("Object"), "Drawer")
+        .AddParameter("expression", _("Level of antialiasing. 2 (LOW), 4 (MEDIUM), or 8 (HIGH). Any other number will be assumed as 2."), "Number");
+  obj.AddCondition("IsAntialiasingOn",
+  _("Is the antialiasing activated for the shape drawer"),
+  _("Checks if the antialiasing is activated for a shape drawer"),
+  _("Is the antialiasing on for _PARAM0_ ?"),
+  _("Setup"),
+  "res/actions/antialiasing24.png",
+  "res/actions/antialiasing.png")
+    .AddParameter("object", _("Object"), "Drawer"); 
+  obj.AddExpression("GetAntialiasingQualityLevel",
```

Comment:
`AddExpressionAndConditionAndAction` can be used.

Reply:
I think I prefer it that way but if it's the standard I'll reimplement it that way ! (I hope this won't break anything)

Reply:
Yes, it's the standard way for new instructions.